src/components/WalletItem.tsx

```diff
@@ -0,0 +1,91 @@
+import React from 'react';
+import {
+  Image,
+  Text,
+  TouchableOpacity,
+  StyleSheet,
+  useColorScheme,
+} from 'react-native';
+import type { Listing } from 'src/types/controllerTypes';
+import { DarkTheme, LightTheme } from '../constants/Colors';
+import { ExplorerUtil } from '../utils/ExplorerUtil';
+
+interface WalletItemProps {
+  currentWCURI: string;
+  walletInfo: Listing;
+}
+
+export const ITEM_HEIGHT = 80;
+
+function WalletItem({ currentWCURI, walletInfo }: WalletItemProps) {
+  const isDarkMode = useColorScheme() === 'dark';
+
+  const onPress = () => {
+    ExplorerUtil.navigateDeepLink(
+      walletInfo.mobile.universal,
+      walletInfo.mobile.native,
+      currentWCURI
+    );
+  };
+
+  return (
+    <TouchableOpacity
+      onPress={onPress}
+      key={walletInfo.id}
+      style={styles.container}
+    >
+      <Image style={styles.icon} source={{ uri: walletInfo.image_url.md }} />
+      <Text
+        style={[styles.name, isDarkMode && styles.nameDark]}
+        numberOfLines={1}
+      >
+        {walletInfo.name}
+      </Text>
+      {/* {walletInfo.isInstalled ? (
```

Comment:
Provide more context as to why it is commented out?

Reply:
Is it related to the `isInstalled` module?

Reply:
i removed that code in #2